app/src/main/res/layout/menu_blocking_switch.xml

```diff
@@ -2,17 +2,56 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<Switch xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The menu will stay open. There are some ideas floating around to make the other part clickable and show some more information about what trackers have been blocked or something like that.